packages/discord.js/typings/index.d.ts

```diff
@@ -5040,7 +5040,12 @@ export interface EmbedField {
   inline: boolean;
 }
 
-export type EmojiIdentifierResolvable = string | EmojiResolvable;
+export type EmojiIdentifierResolvable =
+  | EmojiResolvable
+  | `${string}:${Snowflake}`
```

Comment:
Seeing `a:name:id` is supported according to the jsdoc:
```suggestion
  | `${'' | 'a:'}${string}:${Snowflake}`
```